lib/msal-node/src/config/Configuration.ts

```diff
@@ -129,7 +129,12 @@ export function buildAppConfiguration({
     system,
 }: Configuration): NodeConfiguration {
     return {
-        auth: { ...DEFAULT_AUTH_OPTIONS, ...auth },
+        auth: { ...DEFAULT_AUTH_OPTIONS, 
+            ...auth , 
+            authority: auth.authority?.trim() 
+                ? auth.authority.trim() 
+                : DEFAULT_AUTH_OPTIONS.authority
```

Comment:
Ternary not needed if the condition is also the value:

```suggestion
            authority: auth.authority?.trim() || DEFAULT_AUTH_OPTIONS.authority
```